### ISSUE TITLE
Almost no tests?

### ISSUE THREAD

**Issue by carltonwhitehead** [android10/arrow#8]
It looks like there is almost no test coverage for this project. I gather most of the code was originally sourced from other projects, and so hopefully it'll work, but it would be very reassuring to have more test coverage. Is that on the roadmap?

**Comment by android10** [comment]
@carltonwhitehead yup, you are right. The point is what you mentioned: most of the classes have been grabbed from Guava. 

Always any contribution is possible. I encourage you to start 😃
